### ISSUE TITLE
Feature Request: Add account number to rate-limit response

### ISSUE THREAD

**Issue by benjy3gg** [AsamK/signal-cli#1558]
Could you please add the account number to the rate-limit response?
i am using signal-cli-rest-api in rpc mode and i get the following response when i try to send a message and a rate-limit is encountered:

`{"jsonrpc":"2.0","error":{"code":-5,"message":"Failed to send message due to rate limiting","data":{"response":{"timestamp":1721686852997,"results":[{"recipientAddress":{"uuid":"xxxx-xxxx","number":"+0123456789"},"type":"RATE_LIMIT_FAILURE","token":"xxxx-xxxx-xxxxxyy","retryAfterSeconds":86400}]}}},"id":"xxxxxx-xxyxyyyy"}`

i am using it in multi-number mode, so it would be nice to have the account-number in the response so that i can directly unblock it.

**Comment by AsamK** [comment]
hm, the rate-limit response already includes the "id" field, which corresponds to the id of the send message request.
So using that you should already have the information which account triggered the rate limiting.
